inventory/host_vars/alpha.katuma.org/config.yml

```diff
@@ -1,5 +1,6 @@
 ---
-domain: alpha.katuma.org
+old_domain: alpha.katuma.org
```

Comment:
Would commenting it out be clearer?

Reply:
Yep, I should

Reply:
If is not already there, I would add it to the example with some doc

Reply:
have you seen the doc above? I'm not sure it's worth adding it in example since it's a particularity of Katuma's setup that I'd like others not to repeat. What do you think?